contrib/kafka/cmd/controller/main.go

```diff
@@ -20,6 +21,12 @@ import (
 	"github.com/knative/eventing/pkg/provisioners"
 )
 
+const (
+	// These are Environment variable names.
+	kafkaTLSENnabled = "KAFKA_TLS_ENABLE"
```

Comment:
Can we simply key off the non-empty presence of the `KAFKA_CA_CERT`, to avoid having two knobs (i.e. you don't get TLS even if you set the cert correctly if you forget this setting)?

Reply:
done.